game/src/core/lua_api.h

```diff
@@ -19,4 +19,4 @@ extern "C" {
 #undef MACRO_LUA_GAME
 }
 
-#endif  // CORE_LUA_API_H_
+#endif  // MENGDE_CORE_LUA_API_H_
```

Comment:
@wateret  is it okay to have no namespace?

Reply:
Yes since it is C API.